fetch-pr.py

```diff
@@ -0,0 +1,125 @@
+# given a PR number, get all contributers and the summary from
```

Comment:
Perhaps this could go into the `misc` directory or a new `release-tools` directory